### PR TITLE
Tests for proposals and ratings models

### PR DESCRIPTION
**Pull request by gmmcal**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://gmmcal-tests-proposals-ratings.alagoasdevday-call4papers-3062b296e4351a66c3e5.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://gmmcal-tests-proposals-ratings.alagoasdevday-call4papers-3062b296e4351a66c3e5.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ### Unreleased
 
+* [#34](https://github.com/alagoasdevday/call4papers/pull/34): Create tests for model Rating fixes #24 and closes #24 - [@gmmcal](https://github.com/gmmcal).
```
Comment (pragmaticivan) [inline CHANGELOG.md]: On the next time, please use asc order. So add the new tasks at the end of Unreleased section.

**Comment by pragmaticivan** [comment]
:+1:

File: CHANGELOG.md (hunk 1 above)
Comment (gmmcal) [inline CHANGELOG.md]: I started on desc order. later on I'll update everything to asc ;)

**Comment by pragmaticivan** [comment]
I'm going to accept this one, but please on the next time squash the commits.

**Comment by gmmcal** [comment]
already doing on later PRs. thanks

**Pull request merged**